Content.Shared/Labels/EntitySystems/SharedHandLabelerSystem.cs

```diff
@@ -57,64 +57,92 @@ protected virtual void UpdateUI(Entity<HandLabelerComponent> ent)
     {
     }
 
-    private void AddLabelTo(EntityUid uid, HandLabelerComponent? handLabeler, EntityUid target, out string? result)
+    private void AddLabelTo(EntityUid uid, EntityUid User, HandLabelerComponent? handLabeler, EntityUid target)
     {
         if (!Resolve(uid, ref handLabeler))
         {
```

Comment:
Resolve exists to make a method more versatile, by making it callable both with an Entity and an EntityComponent. Since the component is always provided here, we don't need it